docs/src/create-apps/source-operations.md

```diff
@@ -79,6 +79,9 @@ Replace `<OPERATION_NAME>` with the name of your operation, such as `update-file
 
 {{< /codetabs >}}
 
+After running a source operation, 
+apply the changes to your local development environment by running the `git pull` command.
```

Comment:
:pick: It doesn't feel like a defined command that would be treated like a method, so I'd personally rephrase.
```suggestion
apply the changes to your local development environment by running `git pull`.
```

Reply:
:pick: Or this, though I think it's probably too heavy for a short command.
```suggestion
apply the changes to your local development environment by running the following command: `git pull`.
```